src/services/kratos/auth-email-no-password.ts

```diff
@@ -231,6 +232,35 @@ export const AuthWithEmailPasswordlessService = (): IAuthWithEmailPasswordlessSe
     }
   }
 
+  const loginCookie = async ({
+    email,
+  }: {
+    email: EmailAddress
+  }): Promise<LoginWithPhoneCookieSchemaResponse | KratosError> => {
+    const identifier = email
+    const method = "password"
+    try {
+      const cookieFlow = await createCookieLoginFlow()
+      if (cookieFlow instanceof Error) return cookieFlow
+      const result = await kratosPublic.updateLoginFlow({
+        flow: cookieFlow.flowId,
+        cookie: cookieFlow.cookie,
+        updateLoginFlowBody: {
+          identifier,
+          method,
+          password,
+          csrf_token: cookieFlow.csrf,
+        },
+      })
+      const cookiesToSendBackToClient: Array<SessionCookie> = result.headers["set-cookie"]
+      // note: this only works when whoami: required_aal = aal1
```

Comment:
```suggestion
// identity is only defined when identity has not enabled totp
```

comment is/was inacurate.